README.md

```diff
@@ -39,14 +44,19 @@ Additional for Windows Only:
 
 ## Setup
 
-Clone and `cd` into the repository.
+Clone and `cd` into the repository. Create a file named `.env` and inside it declare
+the variable `SECRET=<secret>` where `<secret>` should be replaced by the passphrase
```

Comment:
I'm not sure SECRET is a good name for an environment variable, it might conflict with other env on some peoples systems. Maybe RPA_TOMORROW_SECRET or something? Should not have any conflicts.

Reply:
True, I renamed it to RPA_SECRET